docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -65,3 +65,19 @@ If you receive the following rule failure: `Bulk Indexing of signals failed: ind
 ====
 If you notice rule delays, review the suggestions above to troubleshoot, and also consider limiting the number of rules that run simultaneously, as this can cause noticeable performance implications in {kib}.
 ====
+
+[discrete]
+[[rule-exceptions-ts]]
+=== Rule exceptions
+
+[discrete]
+[[rule-exceptions-autocomplete-ts]]
+.No autocomplete suggestions
+[%collapsible]
+====
+When creating detection rule exceptions, autocomplete might not provide suggestions in the *Value* field if the data doesn't exist in the current page's time range.
```

Comment:
```suggestion
When you create detection rule exceptions, autocomplete might not provide suggestions in the *Value* field if data doesn't exist in the current page's time range.
```
or maybe "the affected data"? As written I'm not sure what the "the" is referring to

Reply:
or maybe "When you are creating"? As written it seems to say autocomplete is creating detection rules, which might be fine since I'm pretty sure readers will figure out what you actually mean pretty much immediately 🤷

Reply:
Adding "you" to the first part sounds good, natural and conversational. 😄 Throw in a contraction for even more casual tone.

For "the" data, it's the specific data values used to provide autocomplete suggestions, not data in general. Maybe "the values" is more specific, and points back to the **Value** field:

```suggestion
When you're creating detection rule exceptions, autocomplete might not provide suggestions in the *Value* field if the values don't exist in the current page's time range.
```